src/main/java/org/opensearch/security/tools/democonfig/Installer.java

```diff
@@ -356,11 +361,37 @@ static void finishScriptExecution() {
                 System.out.println("### To use the Security Plugin ConfigurationGUI");
             }
 
-            System.out.println("### To access your secured cluster open https://<hostname>:<HTTP port> and log in with admin/admin.");
+            System.out.println(
+                "### To access your secured cluster open https://<hostname>:<HTTP port> and log in with admin/"
+                    + SecuritySettingsConfigurer.ADMIN_PASSWORD
+                    + "."
+            );
             System.out.println("### (Ignore the SSL certificate warning because we installed self-signed demo certificates)");
 
         } catch (Exception e) {
             System.out.println(e.getMessage());
         }
     }
+
+    /**
+     * FOR TESTS ONLY
+     * Resets the class-level variables to their original values
+     * Is needed for tests since the variables are declared static
+     */
+    static void resetState() {
```

Comment:
If Installer was a class in the `static ... main()` method, you could just make an instance of the class, this would allow your tests to create fresh instances without any worry of static state management.

Maintaining this list as new stateful values are added and change behavior will be hard to check

Reply:
okie. I have rewrote the tool to follow singleton pattern.